application/dash_application/table_builder.py

```diff
@@ -82,12 +86,12 @@ def cutoff_table(table_id: str, data: DataFrame, limits: List[Tuple[str, str, fl
                 "if": {"column_id": name, "filter_query": "{%s} contains 'Failed'" % name},
                 "backgroundColor": "mistyrose"
 
-            } for (name, _, _) in limits),
+            } for (name, _, _, _) in limits),
```

Comment:
Can change to `for (name, *_) in limits)` to avoid dealing with element counting